pandas/core/frame.py

```diff
@@ -726,13 +726,32 @@ def style(self):
 
     def iteritems(self):
         """
-        Iterator over (column name, Series) pairs.
```

Comment:
Technically this returns an `Iterator` (or more specifically a `Generator`). So I would leave this as `Iterator` or change this to `Generator`.